src/webui/www/private/index.html

```diff
@@ -6,26 +6,26 @@
     <meta http-equiv="X-UA-Compatible" content="IE=10" />
     <title>qBittorrent ${VERSION} QBT_TR(Web UI)QBT_TR[CONTEXT=OptionsDialog]</title>
     <link rel="icon" type="image/png" href="images/skin/qbittorrent32.png" />
-    <link rel="stylesheet" href="css/dynamicTable.css" type="text/css" />
-    <link rel="stylesheet" type="text/css" href="css/style.css" />
+    <link rel="stylesheet" type="text/css" href="css/dynamicTable.css?v=${VERSION}" />
+    <link rel="stylesheet" type="text/css" href="css/style.css?v=${VERSION}" />
     <!--<link rel="stylesheet" type="text/css" href="css/Content.css" />-->
-    <link rel="stylesheet" type="text/css" href="css/Core.css" />
-    <link rel="stylesheet" type="text/css" href="css/Layout.css" />
-    <link rel="stylesheet" type="text/css" href="css/Window.css" />
-    <link rel="stylesheet" type="text/css" href="css/Tabs.css" />
+    <link rel="stylesheet" type="text/css" href="css/Core.css?v=${VERSION}" />
+    <link rel="stylesheet" type="text/css" href="css/Layout.css?v=${VERSION}" />
+    <link rel="stylesheet" type="text/css" href="css/Window.css?v=${VERSION}" />
+    <link rel="stylesheet" type="text/css" href="css/Tabs.css?v=${VERSION}" />
     <script src="scripts/lib/mootools-1.2-core-yc.js"></script>
     <script src="scripts/lib/mootools-1.2-more.js"></script>
     <!--[if IE]>
-        <script src="scripts/lib/excanvas-compressed.js"></script>
+        <script src="scripts/lib/excanvas-compressed.js?v=${VERSION}"></script>
```

Comment:
3rd party

Reply:
It's third party but not explicitly versioned. It's very unlikely to change, but would still require a version number for cache busting.

Reply:
I have quick look. It seems like "excanvas" is for older IE versions. Do we still need it? Or am I wrong?

Reply:
It looks like it's for IE8 and older. We definitely don't support IE8, so I agree on removing it.